ClientApp/viking-vault/src/ExchangeForm.css

```diff
@@ -0,0 +1,143 @@
+.background-div {
```

Comment:
It doesn't really make sense to use html element names in CSS classes, instead of `.background-div`, why not say `.user-background` or `.background-user` for example

Reply:
Thank you, I've changed it to .exchange-page-background.